raft/src/main/java/org/apache/kafka/raft/EpochState.java

```diff
@@ -31,7 +31,7 @@ default Optional<LogOffsetMetadata> highWatermark() {
      * Decide whether to grant a vote to a candidate.
      *
      * It is the responsibility of the caller to invoke
-     * {@link QuorumState#transitionToVoted(int, ReplicaKey)} if vote is granted.
+     * {@link QuorumState#addVotedState(int, ReplicaKey)} if vote is granted.
```

Comment:
If you agree with my other comment, this should be `QuorumState#transitionToUnattachedVoted`.